cmd/lncli/commands.go

```diff
@@ -887,6 +887,115 @@ func sendPayment(ctx *cli.Context) error {
 	return nil
 }
 
+var sendToRouteCommand = cli.Command{
+	Name:      "sendtoroute",
+	Usage:     "send a payment over predefined routes",
+	ArgsUsage: "(payment_hash $(lncli queryroutes --dest=<dest> --amt=<amt>))",
```

Comment:
I think that the schema of the usage of this command should be the same as defined in the initial issue, or if for some reason you have decided to choose another one, could you explain why?